mmv1/products/accesscontextmanager/terraform.yaml

```diff
@@ -26,11 +26,31 @@ overrides: !ruby/object:Overrides::ResourceOverrides
     skip_sweeper: true
     id_format: "{{name}}"
     import_format: ["{{name}}"]
+    iam_policy: !ruby/object:Api::Resource::IamPolicy
+      exclude: false
+      parent_resource_attribute: 'name'
+      allowed_iam_role: 'roles/accesscontextmanager.policyAdmin'
+      method_name_separator: ':'
+      fetch_iam_policy_verb: :POST
+      import_format: ["accessPolicies/{{name}}", "{{name}}"]
+      iam_conditions_request_type: null
     examples:
       - !ruby/object:Provider::Terraform::Examples
         name: "access_context_manager_access_policy_basic"
         skip_test: true
         primary_resource_id: "access-policy"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "access_context_manager_access_policy_scoped"
```

Comment:
These are 1:1 with the template files so this won't quite work, just one of these is fine.

Reply:
Interesting, the documentation generated just fine. Two examples with the same name but different possible values. Is that not allowed?

Reply:
I've removed the duplicate just so that it won't cause issues when we eventually move the generated tests